packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -51,6 +52,12 @@ class TestRenderingFlutterBinding extends BindingBase with Scheduler, Services,
     if (phase == EnginePhase.paint)
       return;
     renderView.compositeFrame();
+    if (phase == EnginePhase.composite)
```

Comment:
Ah, widget_tester had a slightly different set of phases, I'll reconcile them. Thanks.